build.gradle

```diff
@@ -44,7 +44,7 @@ subprojects {
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '2.9'
+    gradleVersion = '2.13'
```

Comment:
btw, this task configuration is not required any more
Since some 2.x Gradle version, wrapper is a task added to every project and you can update the wrapper with
```bash
./gradlew wrapper --gradle-version 2.0
```